test/lib/specHelperSpec.js

```diff
@@ -0,0 +1,111 @@
+import ActionPrototype from '../../src/lib/ActionPrototype'
+import StorePrototype from '../../src/lib/StorePrototype'
+import Dispatcher from '../../src/lib/ep-dispatcher'
+import assign from 'object-assign'
+import KeyMirror from 'keymirror'
+//import * as FUHelpers from '../support/ComponentHelpers'
+import React from 'react/addons'
+import TestRig from '../../src/testUtils/TestRig.react'
+
+var TestUtils = React.addons.TestUtils;
+
+var TestAction = assign(ActionPrototype, {
+  Types: KeyMirror({
+    GOT_THING: null,
+    NOTHING: null
+  }),
+  fetchThing: function() {
+    //actually fire API
```

Comment:
@BJK Is your plan to add actual calls to the API in tests? What's the reasoning?